src/assets/javascripts/application.js

```diff
@@ -81,6 +81,47 @@ function initialize(config) { // eslint-disable-line func-style
         })
       })
     }
+
+    /* If Clipboard.js is available, inject "copy to clipboard" overlays
+       and attach Clipboard to copy the code.
+       Handle both div.codehilite>pre and pre>code. */
+    // $FlowFixMe
+    if (typeof Clipboard !== "undefined" &&
+        Clipboard.isSupported()) { // eslint-disable-line no-undef
+      const blocks = document.querySelectorAll(
+        "div.codehilite>pre,pre.codehilite>code")
+      let i = 0
+      Array.prototype.forEach.call(blocks, code => {
+        const parent = code.parentNode
+        const codeId = `hl_code${i}`
+        const btn = document.createElement("button")
```

Comment:
I'll have to look into JSX as I've never used it before :slightly_smiling_face:.